docs/advanced/solo-proof-of-stake-voting.md

```diff
@@ -0,0 +1,198 @@
+# Solo Proof-of-Stake (PoS) Voting
+
+## Overview
+Decred’s Proof-of-Stake system requires a user to have a wallet connect to the network in order to purchase tickets so as to participate in its governance model and receive the corresponding rewards. This introduces some risks compared to other setups where a user might choose to store private keys on a paper wallet and not have to worry about a host of attack vectors introduced by being online.
+
+If you want to do something akin to running your own pool, where "hot wallets" on virtual private servers (VPSs) will do the voting on your behalf, but a secure wallet which contains your funds will control ticket buying and receive all rewards this guide is for you.
+
+Basically the advantage of the setup is that the VPSs will be online and ready to vote 24/7 so your machine at home (cool wallet) does not have to be. You will also have redundancy and low latency as you will be running multiple hot wallets on servers in different geographic locations (3 is recommended). Another advantage of this over using a pool is that there is no risk of a pool operator voting in a manner contrary to your wishes, it promotes decentralization, and there is no pool fee. The only additional costs will be those of maintaining the VPSs which may be significantly lower than what a pool might cost if you have a fair number of tickets.
+
+To mitigate a great number of the risks associated with having a Decred wallet connecting to the Internet to purchase tickets a user may want to consider setting up a secure cool wallet (not quite a cold wallet). This guide will first walk you through a low cost solution that can protect the user from several common pitfalls while explaining the rationale behind the recommendations.
+
+---
+
+# Setting Up A Secure "Cool" Wallet
+## Hardware
+The first concept we will address is that of _compartmentalization_. There is really no reason that a machine which may be storing large amounts of cryptocurrency should have multiple roles. It should not be used, for gaming, word processing, web browsing, downloading torrents, etc...
+
+I highly recommend [Qubes OS](https://www.qubes-os.org/) paired with [compatible hardware](https://www.qubes-os.org/hcl/).
+For reference it runs really well on a Thinkpad X260 and even an older Thinkpad X220.
+
+If cost is a huge factor, single-board computers such as the Raspberry Pi 3 Model B can also be tasked for this purpose however using a machine that is more powerful and can handle Qubes is preferable.
+
+---
+
+## Configuration
+Once your staking machine is set up you will need to install the Decred CLI tools.
+1. Import the Decred Release Signing Key in GnuPG.
+
+`gpg --keyserver pgp.mit.edu --recv-keys 0x518A031D`
```

Comment:
Please indent this by four spaces, same for all following bash snippets. This lets mkdocs know that it belongs under the `1.` heading.